scripts/globals/spells/enfeebling_spell.lua

```diff
@@ -449,8 +449,10 @@ xi.spells.enfeebling.useEnfeeblingSpell = function(caster, target, spell)
     -- STEP 5: Exceptions.
     ------------------------------
     -- Bind: Dependant on target speed.
+    -- Bind: Duration floor of 5 seconds.
     if spellEffect == xi.effect.BIND then
         potency = target:getSpeed()
+        duration = utils.clamp(duration, 5, 60)
```

Comment:
this probably makes more sense on line 259 which is already clamping from 1-60?

Reply:
I went that way originally but later realized we would be skewing the duration slightly longer  by doing so. Since the result of line 259 is later multiplied by the resistance factor which will be between 0.5-1, to achieve an end result floor of 5, we'd have to clamp 259 between 10-60. That's my reasoning anyhow. Happy to clamp on 259 though if it wouldn't affect the distribution too much.

Reply:
Do we know if getting 1/2 resisted puts the floor below 5 seconds? If not, this is the appropriate place

Reply:
I believe it does. My first brush, I did utils.clamp(duration, 5, 60) on line 259 as suggested but in my local testing i observed a 2-3 second bind duration.

Reply:
the linked capture data does should some binds at 3-4 seconds so I would assume the clamp would need to happen before the resistance check. the original PR data even shows a 1s bind which I presume is a 1/8th resist or something

Reply:
The original PR #6194 was implemented based off of retail capture data and did not include any floor but did establish the mean and standard deviation. 
The purpose of this PR is to add the floor, as SE changed bind many times over the years- durations, floors, even the name of the spell. The floor of 5s is based on the video capture data from toau era. In that set the shortest bind observed is 7s. It seemed more reasonable 5s would be the floor than 7.

Reply:
good enough for me